packages/server/src/service/letter/index.ts

```diff
@@ -106,7 +106,8 @@ export class Letter {
         confirmationId: this.confirmationId,
         method: {
           ...method,
-          faxes: method.faxes && method.faxes.map(formatPhoneNumber),
+          // If method.faxes is already a dict, then this is a resend, and faxes are already in a good format.
+          faxes: method.faxes && (method.faxes.constructor === Array ? method.faxes.map(formatPhoneNumber) : method.faxes),
```

Comment:
When I revert this line, I get `(node:13831) UnhandledPromiseRejectionWarning: TypeError: method.faxes.map is not a function`

When I inspect the value of method.faxes, it is `{ '0': '+17182465958' }`

I have no idea how this is possible, but will try to investigate.  In the mean time, I think the PR is otherwise in a good state.

Reply:
I could also just call .values() if we detect it's not an array.

Note that this dict that I see when I inspect the value of method.faxes is the same as what's in Firestore (at least for the documents I'm using, and I never overwrote the value of method.faxes there).